src/libgeodecomp/io/logger.h

```diff
@@ -18,6 +18,10 @@ namespace LibGeoDecomp {
  * happens at compile time, thus no runtime overhead is incurred for
  * filtered messages.
  */
+#ifdef ERROR
```

Comment:
I think there was a time when we did, yes, but IIRC @sithhell got rid of this. :-)